src/components/landingPage/articleCard/ratingComponent/index.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import Rating from 'react-rating';
+import style from './ratingComponent.scss';
+
+const theme = {
+  color: '#ffd700',
+  border: '1px solid #fff',
+  fontSize: '13px',
+};
+export default class RatingArticleCardComponent extends Component {
```

Comment:
The rating component should be put in the common components to allow others reuse it

Reply:
I worked on the feedback. It is in common components